stepman/paths.go

```diff
@@ -200,6 +200,12 @@ func GetLibraryBaseDirPath(route SteplibRoute) string {
 	return path.Join(GetCollectionsDirPath(), route.FolderAlias, "collection")
 }
 
+// GetStepCollectionSpecPath ...
+// Location of steplib.yml of the collection marked by this route
```

Comment:
I guess you wanted to write these in a single sentence.

Reply:
It's interesting, all the other methods have this format 🤔

Reply:
I think they are messed up as well :D We usually use `<functionname> ...` to get pass golint. But you can give proper comments to exposed functions.